ts/Extensions/Pane/PaneComposition.ts

```diff
@@ -123,32 +123,38 @@ function isInsidePane(
 
     if (defined(startAngle) && defined(endAngle)) {
         // Round angle to N-decimals to avoid numeric errors
-        const angle = Math.atan2(
+        let angle = Math.atan2(
             correctFloat(y - cy, 8),
             correctFloat(x - cx, 8)
         );
 
+        // Normalize angle to [0, 2π)
+        angle = (angle + 2 * Math.PI) % (2 * Math.PI);
```

Comment:
Instead of calculating `2 * Math.PI` multiple times, let's just calculate it once